Libraries/LibWeb/IndexedDB/Internal/Algorithms.cpp

```diff
@@ -36,9 +36,16 @@ WebIDL::ExceptionOr<GC::Ref<IDBDatabase>> open_a_database_connection(JS::Realm&
 
     // 2. Add request to queue.
     queue.append(request);
+    dbgln_if(IDB_DEBUG, "open_a_database_connection: added request {} to queue", request->uuid());
 
     // 3. Wait until all previous requests in queue have been processed.
     HTML::main_thread_event_loop().spin_until(GC::create_function(realm.vm().heap(), [queue, request]() {
+        dbgln_if(IDB_DEBUG, "open_a_database_connection: waiting for step 3");
+        dbgln_if(IDB_DEBUG, "requests in queue:");
+        for (auto const& item : queue) {
+            dbgln_if(IDB_DEBUG, "[{}] - {} = {}", item == request ? "x"sv : " "sv, item->uuid(), item->processed() ? "processed"sv : "not processed"sv);
+        }
```

Comment:
When you have multiple lines like this, it can be nicer to wrap them in `if constexpr (IDB_DEBUG) {}` instead, and then use regular `dbgln()` inside. That also means the loop wouldn't run if the debug flag is disabled.

Same for a couple of other places in this commit.